index.html

```diff
@@ -0,0 +1,101 @@
+        <meta name="viewport" content="width=device-width" />
+        <link rel="stylesheet" href="myStyles.css" type="text/css"></link>
+
+        <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+        <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+        <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+
+        <title>My Portfolio</title>
+    </head>
+    <body>
+        <nav id="topBar" class="topBar">
+            <!--Status-->
+            <section class="status_TopNav">
+                <!--Hora-->
+                <div class="timeContainer">
+                    <a id="homeTime" href="javascript:void(0);">
+                        <img id="timeImage" src="images/Status/Time.png" alt="timeAncor">
+                    </a>
+                </div>
+                <!--Status-->
+                <div class="statusContainer">
+                    <a id="homeStatus" href="javascript:void(0);">
+                        <img id="statusImage" src="images/Status/Status.png" alt="statusAncor">
+                    </a>
+                </div>
+            </section>
+            <!--Nav bar-->
+            <section class="menuAngosto_TopNav">
+                <!--Logo-->
+                <div class="logoAngostoContainer">
+                    <a id="homeLogo_Angosto" class="homeLogo_Angosto">
+                        My Logo
+                    </a>
+                </div>
+                <!--Menu Hamburger-->
+                <div class="menuHamburgerContainer">
+                    <a id="menuHamburger" href="javascript:void(0);" class="icon">
+                        <img id="baraMenuImage" src="images/barMenu.png" alt="menuAncor">
+                    </a>
+                </div>
+            </section>
+        </nav>
+        <section>
+            <div class="mainPage">
+                <div class="flexContainer">
+                    <!--Links a redes sociales Top-->
+                    <div class="socialLinks_Top">
+                        <a href="javascript:void(0);">
+                            <img id="gatoIconTop" src="images/Iconos/Gato.png" alt="iconGato">
+                        </a>
+                        <a href="javascript:void(0);">
+                            <img id="linkedinIconTop" src="images/Iconos/Linkedin.png" alt="iconLinkedin">
+                        </a>
+                        <a href="javascript:void(0);">
+                            <img id="manitoIconTop" src="images/Iconos/Manito.png" alt="iconManito">
+                        </a>
+                        <a href="javascript:void(0);">
+                            <img id="twitterIconTop" src="images/Iconos/Twitter.png" alt="iconTwitter">
+                        </a>
+                        <a href="javascript:void(0);">
+                            <img id="mIconTop" src="images/Iconos/M.png" alt="iconM">
+                        </a>
```

Comment:
- [ ] Kindly remove this section from your codebase as it is not part of the project requirement. what do you think?